components/form_inputs/autocompletes/AutocompleteStyle.vue

```diff
@@ -0,0 +1,191 @@
+                    label="Basic"
+            ></v-autocomplete>
+          </v-col>
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    filled
+                    label="Filled"
+            ></v-autocomplete>
+          </v-col>
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo"
+                    solo
+            ></v-autocomplete>
+          </v-col>
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo-inverted"
+                    solo-inverted
+            ></v-autocomplete>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-card>
+
+    <h3>Dense</h3>
+    <v-card>
+      <v-container>
+        <v-row
+                align="center"
+        >
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Shaped outlined"
+                    dense
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Shaped outlined"
+                    dense
+                    shaped
+                    outlined
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    dense
+                    rounded
+                    filled
+                    label="Filled rounded"
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    dense
+                    filled
+                    outlined
+                    label="Filled outlined"
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo rounded"
+                    dense
+                    rounded
+                    solo
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo-inverted outlined"
+                    dense
+                    outlined
+                    solo-inverted
+            ></v-autocomplete>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-card>
+
+    <h3>Dark</h3>
+    <v-card dark>
+      <v-container>

```

Comment:
props 'dark' should be in v-autocomplete. Not in v-card. Please, correct. It works in different way in v-autocomplete.

Reply:
There's no real difference in output because v-card passes _dark_ attribute to underlying tags elements, though i can add it specifically to every element if you insist. And card should be dark to give dark background to elements

Reply:
Then maybe it would be useful to add such comment so that the tester coming after us would not look for _v-autocomplete--dark_If I were not involved into updating test-site I wouldn't guess such nuinces.